src/Providers/Storage/Illuminate.php

```diff
@@ -11,13 +11,13 @@
 
 namespace Tymon\JWTAuth\Providers\Storage;
 
-use Illuminate\Cache\CacheManager;
 use Tymon\JWTAuth\Contracts\Providers\Storage;
+use Illuminate\Contracts\Cache\Store as StoreContract;
```

Comment:
Sorry about that, send a fix to #271

> In my opinion laravel Managers are so flexible, so there's no need to use contracts in this case

It either go with a contract that can be resolved with IoC (AuthManager and CacheManager is not aliased in Lumen Framework, unlike Laravel https://github.com/laravel/lumen-framework/blob/5.1/src/Application.php#L1656-L1659), or swap the way jwt packages resolve the providers class.